lib/pages/admin/delete_choice_board_category.dart

```diff
@@ -1,54 +1,52 @@
 import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:firebase_storage/firebase_storage.dart';
 import 'package:flutter/material.dart';
 
+import '../../helpers/firebase_functions.dart';
+
 /// Deletes a choiceboard category given ID
 class DeleteChoiceBoardCategory extends StatelessWidget {
   final String categoryId;
-
-  const DeleteChoiceBoardCategory({Key? key, required this.categoryId})
-      : super(key: key);
+  final String categoryName;
+  late final FirebaseAuth auth;
+  late final FirebaseFirestore firestore;
+  late final FirebaseStorage storage;
+  late final FirebaseFunctions firestoreFunctions;
+
+  DeleteChoiceBoardCategory({super.key, required this.categoryId, required this.categoryName, FirebaseAuth? auth, FirebaseFirestore? firestore, FirebaseStorage? storage}) {
+    this.auth = auth ?? FirebaseAuth.instance;
+    this.firestore = firestore ?? FirebaseFirestore.instance;
+    this.storage = storage ?? FirebaseStorage.instance;
+    firestoreFunctions = FirebaseFunctions(auth: this.auth, firestore: this.firestore, storage: this.storage);
+  }
 
   @override
   Widget build(BuildContext context) {
     // Return to admin screen if user cancels choice
     Widget cancelButton = TextButton(
-      child: Text("No"),
+      child: Text("Cancel"),
       onPressed: () {
         Navigator.pop(context);
       },
     );
 
-    // Delete document from firebase collection
-    void deleteFromCollection(String collectionName) {
-      FirebaseFirestore.instance
-          .collection(collectionName)
-          .doc(categoryId)
-          .delete()
-          .then(
-            (doc) => print("Document deleted"),
-            onError: (e) => print("Error updating document $e"),
-          );
-    }
-
     // Once user confirms choice, call delete function
-    Widget confirmButton = TextButton(
-      child: Text("Yes"),
-      onPressed: () {
-        // Delete associated document from 'categoryItems' collection
-        deleteFromCollection("categoryItems");
-
-        // Delete category from 'categories' collection
-        deleteFromCollection("categories");
+    Widget deleteButton = TextButton(
+      child: Text("Delete"),
```

Comment:
Force this button to be red, so it becomes consistent

Reply:
I think I was told to remove hardcoded button styling? Though I will re-add the colouring in the mean time